code_samples/api/rest_api/load_content.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+// cURL
+
+$resource = 'https://api.example.com/api/ibexa/v2/content/objects/52';
+$curl = curl_init($resource);
+curl_setopt_array($curl, [
+    CURLOPT_HTTPHEADER => ['Accept: application/vnd.ibexa.api.ContentInfo+json'],
+    CURLOPT_HEADERFUNCTION => static function ($curl, $header) {
+        if (!empty($cleanedHeader = trim($header))) {
+            var_dump($cleanedHeader);
+        }
+
+        return strlen($header);
+    },
+    CURLOPT_RETURNTRANSFER => true,
+]);
+var_dump(json_decode(curl_exec($curl), true));
```

Comment:
We should just drop cURL examples altogether.Symfony HttpClient and Guzzle are so popular nowadays I just can't imagine anyway wanting to even bother with raw cURL calls 😉

Reply:
@webhdx 
Because you need nothing else than PHP to run this. You can copy-paste that in `php -a` without having to be at the root of a Symfony project having the HTTP Client bundle.

But, this is highly questionable. Who read the Ibexa Developer Documentation without having an Ibexa DXP project available but only PHP??

You're right, I'll remove this.

Reply:
@webhdx 
I removed cURL example from "Testing REST API" "PHP" section in
616622550b7044dcbc0682f4902d4fe6df1df1a9...ab4926d330ed65c3bc7f0614143720d77e4db241
I kept the headers dumping like in the JS example for the same reason, an opportunity to show REST response headers `accept-patch` and `content-type`.